src/components/Tag/PTag.vue

```diff
@@ -37,29 +37,39 @@
   const tagValue = computed(() => {
     return normalize(props.value ?? props.label ?? '')
   })
+
+  const classes = computed(() => ({
+    root: {
+      'p-tag--dismissible': props.dismissible,
+    },
+  }))
 </script>
 
 <style>
 .p-tag { @apply
-  gap-1
+  gap-2
   inline-flex
   items-center
-  rounded-full
-  bg-background-400
-  text-foreground
-  dark:bg-background-600
+  rounded-md
   px-2.5
   py-0.5
-  text-sm
+  text-sm;
+  background-color: var(--p-color-tag-bg);
+  border-color: var(--p-color-tag-border);
+  color: var(--p-color-tag-text);
 }
 
-.p-tag__icon { @apply
+.p-tag__icon,
+.p-tag__icon svg { @apply
```

Comment:
Seems like this `svg` selector shouldn't be needed if `p-icon` is working properly. Might be worth looking into.

Reply:
Yeah that's what I had figured too but didn't dig in, wasn't working properly in the tags demo. I'll go ahead and scope it out before merging this.

Reply:
So `p-icon` didn't have any sort of max width/height. I went ahead and added those in for that component. We'll just want to keep an eye out for any icons that get squeezed in the UI after the update.

Reply:
Ah you know what though.. Maybe `p-icon` shouldn't have a max width/height like that but enforce it's standard sizes.. `p-tag` is setting a width and height when it should just be setting the icon to `size="small"`. I'll update that real quick in the rebrand branch directly.